IdentityCore/src/webview/response/MSIDCBAWebAADAuthResponse.m

```diff
@@ -58,6 +58,7 @@ - (instancetype)initWithURL:(NSURL *)url
     {
```

Comment:
This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.
<!-- changelog-comment -->